### PR TITLE
Global configuration option

### PR DESCRIPTION
**Pull request by rs4231199**
Added: A global configuration option to enable/disable inline editing

image

### CODE REVIEW

**Comment by rs4231199** [comment]
@roland-d Please review this PR.

File: installation/configuration.php-dist [hunk 1]
```diff
@@ -36,6 +36,7 @@ class JConfig
 	public $list_limit = 20;
 	public $access = 1;
 	public $frontediting = 1;
+	public $frontinlineediting = true;
```
Comment (roland-d) [inline installation/configuration.php-dist]: @rs4231199 This default value should be false as it is also false in your XML file. The default value of false is the correct one.

**Comment by rs4231199** [comment]
@roland-d 
Is there any file that I skipped? Or is it good to merge?

**Comment by roland-d** [comment]
@rs4231199 It is good to go codewise. Just need to update this documentation page: https://docs.joomla.org/Help4.x:Site_Global_Configuration 

Just no idea when we need to do that or where to write that down. Do you keep a list of documentation to update? If so, add this. If  not, start it and add this :)

**Comment by rs4231199** [comment]
I have planned to add functionality to quickly and temporarily disable inline editing by adding a URL parameter(for example, `?noinline=1`). What are your thoughts on this?
@roland-d @bembelimen @anibalsanchez

**Comment by roland-d** [comment]
I do not think we should add that. One setting for a feature is enough.

**Comment by anibalsanchez** [comment]
Agreed. Less is more.

**Comment by rs4231199** [comment]
👍 I will merge it now.

**Pull request merged**